corehq/apps/commtrack/util.py

```diff
@@ -174,10 +174,9 @@ def submit_mapping_case_block(user, index):
         )
 
     submit_case_blocks(
-        ElementTree.tostring(
-            caseblock.as_xml()
-        ),
+        ElementTree.tostring(caseblock.as_xml()),
```

Comment:
Not your code I know but this is the same as `caseblock.as_string()`. A few other places too.